_topic_maps/_topic_map_ms.yml

```
@@ -138,7 +138,7 @@ Name: Running applications
 Dir: microshift_running_apps
 Distros: microshift
 Topics:
- - Name: Running applications
+ - Name: Operators
```

This isn't just about operators.

```suggestion
 - Name: Deploying workloads
```